go/pkg/rbac/permissions.go

```diff
@@ -62,6 +62,9 @@ const (
 
 	// ReadKey permits viewing API key details
 	ReadKey ActionType = "read_key"
+
+	// VerifyKey permits verifying API keys
+	VerifyKey ActionType = "verify_key"
```

Comment:
@coderabbitai please create an issue for us . we need to create this verification type in the dashboard too, otherwise users won't be able to create this permission